src/changelog/sources/renovate/source_test.go

```diff
@@ -41,9 +41,8 @@ func (c *commitsGetterMock) Commits(_ string) ([]git.Commit, error) {
 	return commits, nil
 }
 
-//nolint:funlen
```

Comment:
Why are we getting rid of `paralleltest`? If this is unavoidable I'd love to see a short explanation.